### DISCUSSION TITLE
How to hide hierarchies?

### DISCUSSION THREAD

**Discussion by agent3133** [atoti/atoti#563] [Q&A] [answered]
Hi there,

there is a property ".visible" on hierarchies, but it doesn't hide the hierarchy from the UI, as similar property does on the measure. See example below:

![image](https://user-images.githubusercontent.com/24815479/163393022-b891ccdf-06fa-46d2-90db-61001e266d3a.png)

```py
import atoti as tt
import pandas as pd 

session = tt.Session()
session.link()
df = pd.DataFrame({"TradeId":["1","2","3"], 
                   "Cashflows":[100, 200, 300], 
                   "Customer": ["Homer", "Bart", "Marge"]}
                   )
table = session.read_pandas(df, table_name="Hide Hirarchy Example")
cube = session.create_cube(table)

cube.hierarchies["TradeId"].visible = False
cube.measures["Cashflows.MEAN"].visible = False

session.visualize()
```

**Comment by bloussou** [comment, marked as answer]
Duplicate of https://github.com/atoti/atoti/issues/466, I'm reopening the ticket as it is indeed not working.

**Comment by bloussou** [comment]
Hello,
This will be fixed in the next release.
